src/admin-views/admin-update-message.php

```diff
@@ -6,8 +6,11 @@
 
 ?>
 
-/* translators: the placeholder will contain the semantic version of The Events Calenda plugin, e.g. 1.2.3 */
-<p class="tribe-update-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
+
+<p class="tribe-update-message"><?php
+	/* translators: the placeholder will contain the semantic version of The Events Calenda plugin, e.g. 1.2.3 */
+	echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) );
+	?></p>
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected 0 tabs, found 1
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: ff4235a0d91a0625387821900181cc47</sub>